deploy/cephfs/kubernetes/v1.13/csi-provisioner-rbac.yaml

```diff
@@ -3,6 +3,8 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: cephfs-csi-provisioner
+  # replace with non-default namespace name
+  namespace: default
```

Comment:
for namespace use `namespace: {{ .Release.Namespace }}`  applies for other places also